tests/test_manager.py

```diff
@@ -33,52 +38,69 @@ def test_entity_reference_valid(manager, printer):
 
 
 def test_entity_reference_exists(project, manager):
-
     project: ProjectInfo
-    result = manager.entityExists(
+    context = manager.createContext()
+
+    # TODO(DF): OpenAssetIO beta.1 is lacking convenience overloads for
+    #  entityExists, so we must use the lower-level callback-based
+    #  signature, for now.
+
+    results = [None, None]
+
+    manager.entityExists(
         [
-            (
+            manager.createEntityReference(
                 f"ayon+entity://{project.project_name}/"
                 f"{project.folder.name}?product={project.product.name}&"
                 f"version={project.version.name}&"
                 f"representation={project.representation.name}"
             ),
-            (
+            manager.createEntityReference(
                 f"ayon+entity://{project.project_name}/"
                 f"{project.folder.name}?product={project.product.name}&"
                 f"version={project.version.name}&"
                 f"representation=NOT_EXISTING",
             )
-        ], None)
+        ],
+        context,
+        lambda idx, result: operator.setitem(results, idx, result),
+        raise_batch_element_error)
 
-    assert result[0] is True
-    assert result[1] is False
+    assert results[0] is True
+    assert results[1] is False
 
 
 def test_resolve(project, manager):
     project: ProjectInfo
     context = manager.createContext()
 
-    result = manager.resolve(entityReferences=[
-            openassetio.EntityReference(
-                (
-                    f"ayon+entity://{project.project_name}/"
-                    f"{project.folder.name}?product={project.product.name}&"
-                    f"version={project.version.name}&"
-                    f"representation={project.representation.name}"
-                )),
-        ],
+    result = manager.resolve(
+        entityReference=manager.createEntityReference(
+            f"ayon+entity://{project.project_name}/"
+            f"{project.folder.name}?product={project.product.name}&"
+            f"version={project.version.name}&"
+            f"representation={project.representation.name}"
+        ),
         traitSet={mc_traits.content.LocatableContentTrait.kId},
+        resolveAccess=openassetio.access.ResolveAccess.kRead,
         context=context)
 
-    # result: List[openassetio.TraitsData]
-    assert result[0].hasTrait(mc_traits.content.LocatableContentTrait.kId)
-    assert result[0].getTraitProperty(
-        mc_traits.content.LocatableContentTrait.kId,
-        "location") == os.path.normpath(
-            f"C:/projects/{project.project_name}/"
-            f"{project.folder.name}/publish/render/"
+    locatable_content = mc_traits.content.LocatableContentTrait(result)
+
+    assert locatable_content.isImbued()
+
+    assert locatable_content.getLocation() == (
+            f"file://{project.project_root_folders[platform.system().lower()]}/"
```

Comment:
Not yet tested that this string interpolation works on Windows

Reply:
Tested on Windows, found an issue with this string interpolation (missing `/` prefix for Windows paths).  Force-pushed a fix.

So tests are now passing on Windows and Linux 👍